<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Report</class>
 <widget class="QWidget" name="Report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="QuarBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>季度报表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YearBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>年度报表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MonthBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>月报表</string>
   </property>
  </widget>
  <widget class="QComboBox" name="YearCBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2018年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019年</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="InerLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>切换用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>TitleLab</zorder>
  <zorder>InerLab</zorder>
  <zorder>QuarBtn</zorder>
  <zorder>YearBtn</zorder>
  <zorder>MonthBtn</zorder>
  <zorder>YearCBox</zorder>
  <zorder>label</zorder>
  <zorder>CloseBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
